### PR TITLE
New contributions

### PR DESCRIPTION
**Pull request by ScottSoren**
A few new things here, all having to do with MS quantification, which @spoeka22 and @AlexanderKrabbe contributed to the user_ready branch while @KennethNielsen and I worked on #30:

- functionality in `MSInlet.gas_flux_calibration` for calibration with dilute calibration gas 
- an effective capillary length calculator in MSInlet
- CO2 properties added to constants.py

It's been tested against my most advanced EC-MS calibration script, https://github.com/ScottSoren/pyCOox_public/blob/ixdat_v0p2/paper_I_fig_S1/paper_I_fig_S1.py

*I think we can close the **user_ready** branch when this is merged to **main**!*

### CODE REVIEW

File: src/ixdat/techniques/ms.py [hunk 1]
```diff
@@ -464,15 +464,56 @@ def __init__(
...
         self.T = T
         self.gas = gas  # TODO: Gas mixture class. This must be a pure gas now.
 
+    def calc_l_cap_eff(
+        self, n_dot_measured, gas=None, w_cap=None, h_cap=None,  T=None, p=None):
+        """Calculate gas specific effective length of the capillary in [m]
+            and add {gas:value} to l_cap_eff (dict)
+
+        Args:
+            w_cap (float): capillary width [m], defaults to self.w_cap
```
Comment (KennethNielsen) [inline src/ixdat/techniques/ms.py:480]: Capitals after colon. There are a few of these

File: src/ixdat/techniques/ms.py [hunk 2]
```diff
@@ -464,15 +464,56 @@ def __init__(
...
+
+        n_dot_predicted = self.calc_n_dot_0(gas=gas, w_cap=w_cap, h_cap=h_cap, T=T, p=p)
+
+        l_cap_gas_specific_eff = self.l_cap * n_dot_predicted / n_dot_measured
+        self.l_cap_eff[gas] = l_cap_gas_specific_eff #add effective l_cap for specific gas
+
+        return l_cap_gas_specific_eff
+
+    def update_l_cap(self, gases=[]):
+        """ Update self.l_cap from average of values in dict l_cap_eff
```
Comment (KennethNielsen) [inline src/ixdat/techniques/ms.py:498]: No space before "update"

**Review by KennethNielsen — APPROVED** [review]
A few minor docstring nitpicks. Otherwise good to go from me. I'll set to approve as they are so small.

**Comment by AlexanderKrabbe** [comment]
I will tidy up the docstring in a new pull request when I figure out my git again.

**Comment by KennethNielsen** [comment]
@AlexanderKrabbe that is a little fun since Scott kind of submitted this one for you in this extra round. The branch that we are merging is owned by this main repository, which you don't have push access for. If you want to fix it, what I would suggest is that you make yourself a new feature branch, of this one ixdat/ixdat:new_contributions fix stuff there and then ask Scott to merge the fixes into this branch.

**Comment by AlexanderKrabbe** [comment]
@KennethNielsen ahhh. smart :-)

**Comment by KennethNielsen** [comment]
@AlexanderKrabbe I hope so. Let me know if the git gives you gripes.

**Comment by ScottSoren** [comment]
@AlexanderKrabbe addressed the comments in #48. Merging!

**Pull request merged**